lib/galaxy/webapps/galaxy/controllers/user.py

```diff
@@ -563,6 +563,19 @@ def __validate_login( self, trans, **kwd ):
                 message, status = self.resend_verification_email( trans, user.email, user.username )
         else:  # activation is OFF
             message, success, status = self.proceed_login( trans, user, redirect )
+            pw_expires = trans.app.config.password_expiration_period
+            if pw_expires:
+                if user.last_password_change < datetime.today() - pw_expires:
+                    trans.response.send_redirect(web.url_for(controller='user',
+                                                             action='change_password',
+                                                             message='Your time is up! Change your password to access galaxy',
```

Comment:
Please rephrase. Maybe something like `Your password has expired. Please change it to access Galaxy.`